frontend/src/Login/Login.js

```diff
@@ -17,25 +17,25 @@ class Login extends React.Component {
         e.preventDefault();
         const form = e.target;
         const user = {
-            username: form.elements['username'].value,
+            employeeID: form.elements['username'].value,
             password: form.elements['password'].value
         };
-        this.setState({
-            isVisible: false,
-            userName: 'Shekhar'
-        });
-        /*axios.get(`/validateLogin/${user}`)
+        // this.setState({
+        //     isVisible: false,
+        //     userName: 'Shekhar'
+        // });
+        axios.post(`/user/validateLogin`, user)
             .then(res => {
                 this.setState({
-                    isVisible: true,
-                    userName: 'Shekhar'
+                    isVisible: res.data.employeeName,
```

Comment:
Is it `res.data.employeeName` for isVisible flag ?

Reply:
yep.. if the login is valid it will set a employee name or else it wont.. So it would come up as false